crates/task-impls/src/network.rs

```diff
@@ -501,79 +607,123 @@ impl<
             }
         });
     }
+}
 
-    /// handle `VidDisperseSend`
-    async fn handle_vid_disperse_proposal(
-        &self,
-        vid_proposal: Proposal<TYPES, VidDisperse<TYPES>>,
-        sender: &<TYPES as NodeType>::SignatureKey,
-    ) -> Option<HotShotTaskCompleted> {
-        let view = vid_proposal.data.view_number;
-        let vid_share_proposals = VidDisperseShare::to_vid_share_proposals(vid_proposal);
-        let mut messages = HashMap::new();
+/// A module with test helpers
+pub mod test {
```

Comment:
i dont know how much of a pain itll be / if it makes sense to somehow move this outside this file.